Formula 1/Views/Cells/CircuitCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  CircuitCell.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CircuitCell: UICollectionViewCell {
```

Comment:
```suggestion
final class CircuitCell: UICollectionViewCell {
```

Reply:
whats the point of final here?

Reply:
Compiler optimizations: https://developer.apple.com/swift/blog/?id=27

Reply:
Same as the previous link